Lib/importlib/_bootstrap_external.py

```diff
@@ -586,7 +586,7 @@ def _code_to_timestamp_pyc(code, mtime=0, source_size=0):
     "Produce the data for a timestamp-based pyc."
     data = bytearray(MAGIC_NUMBER)
     data.extend(_pack_uint32(0))
-    data.extend(_pack_uint32(mtime))
+    data.extend(_pack_uint32(mtime) & 0xFFFF_FFFF)
```

Comment:
This change is useless, _pack_uint32() already does "& 0xFFFFFFFF". importlib can be left unchanged.